### ISSUE TITLE
Use UIA for PowerPoint

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#10430]
### Is your feature request related to a problem? Please describe.
Text controls in PowerPoint are using the PowerPoint object model to be read. From what I've seen from enforcing UIA in PowerPoint, the UIA implementation is pretty OK.

### Describe the solution you'd like
Consider using UIA for PowerPoint, at least in text controls but may be even for the GUI that allows you to create presentations.

### Describe alternatives you've considered
Leave as is.

### Additional context
May be @michaelDCurran knows how the UIA implementation in PowerPoint is related to the one in Word?

**Comment by lukaszgo1** [comment]
Are there any advantages of doing this  other than the fact that UIA is more modern?

**Comment by leonardder** [comment]
Quick testing revealed tat you can use proper spell checking in 
Powerpoint when using UIA. I assume there's more stuff exposed using UIA 
that we don't use in the current implementation. I'd also imagine that 
it is slightly faster than the object model implementation.

**Comment by zstanecic** [comment]
Hi to all,

All UIA implementations for MS office are basicallz faster in practice. 

The problem is that these are not unified, and noti n all situations work ok.

**Comment by codeofdusk** [comment]
Any updates on this issue? Cc @michaelDCurran.

**Comment by michaelDCurran** [comment]
NV Access has no plans to work on this at the moment. However, If it seems that there is an increase in users who need to run Powerpoint in WDAG (Defender Application Guard) and or RAIL (Remote) then we will reconsider the priority on this.

Other than the scenarios where the object model is unavailable (such as above), I don't believe that switching to UIA will provide any significant bennifit to Powerpoint. The amount of info we need to query in Powerpoint is significantly less than Word / Excel.

**Comment by LeonarddeR** [comment]
@SaschaCowley Given recent research in PowerPoint, has this been considered?
For example, rather than relying on the object model for text frames, it looks like UIA is pretty accurate as well.

**Comment by SaschaCowley** [comment]
@LeonarddeR Not really. The focus has been on trying to iron out some Braille issues, but I will bring this up with the team. My suspicion is that https://github.com/nvaccess/nvda/issues/10430#issuecomment-849200821 still stands.
